### DISCUSSION TITLE
Add OCI regions : Singapore 2, Riyadh, Jovanovac

### DISCUSSION THREAD

**Discussion by samsit88** [VarunAgw/CloudPingTest#48] [Q&A] [answered]
Could you add Oracle OCI regions:
- ap-singapore-2 (XSP)
- me-riyadh-1 (RUH)
- eu-jovanovac-1 (BEG)

**Comment by VarunAgw** [comment, marked as answer]
Added all 3 regions - https://cloudpingtest.com/oracle
